spec/models/aroc_week_1_spec.rb

```diff
@@ -42,11 +41,12 @@
 
   it '3. finds order id of largest order' do
     # ----------------------- Using Raw SQL ----------------------
-    order_id = ActiveRecord::Base.connection.execute('SELECT id FROM orders ORDER BY amount DESC LIMIT 1').first['id']
+    # order_id = ActiveRecord::Base.connection.execute('SELECT id FROM orders ORDER BY amount DESC LIMIT 1').first['id']
     # ------------------------------------------------------------
 
     # ------------------ Using ActiveRecord ----------------------
     # Solution goes here
+    order_id = Order.select("orders.*").order("amount desc").limit(1).first[:id]
```

Comment:
In this case, you don't need `select("orders.*")`. Since you start the query with `Order`, AR will assume you want `orders.*` returned.